bundles/org.openhab.binding.solarwatt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solarwatt"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="energymanager">
+		<label>Solarwatt Energymanager</label>
+		<description>Solarwatt Energymanager is the bridge to all things attached to the PV production system.</description>
+
+		<representation-property>hostname</representation-property>
+
+		<config-description>
+			<parameter name="hostname" type="text">
+				<label>Host Name / IP Address</label>
+				<description>The host name/ip address of the solarwatt energymanager.</description>
+				<context>network-address</context>
+				<advanced>false</advanced>
+			</parameter>
+			<parameter name="refresh" type="integer" unit="s">
+				<label>Refresh Data Period</label>
+				<description>Period between updates to the devices data in seconds.
+				</description>
+				<default>30</default>
+				<unitLabel>s</unitLabel>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="rescan" type="integer" unit="min">
+				<label>Redetect Devices Period</label>
+				<description>Period between updates to the detected devices in minutes.
+				</description>
+				<default>5</default>
+				<unitLabel>min</unitLabel>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="evstation">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="energymanager"/>
+		</supported-bridge-type-refs>
+
+		<label>EV Station</label>
+		<description>Electric vehicle charger station</description>
+
+		<representation-property>IdName</representation-property>
+
+		<config-description>
```

Comment:
You could define the config paramater once and reuse it for all Thing types. See https://github.com/openhab/openhab-addons/blob/503abab1819e5db3da3678f780e8086b7734b72b/bundles/org.openhab.binding.lcn/src/main/resources/OH-INF/config/config.xml#L8 and https://github.com/openhab/openhab-addons/blob/503abab1819e5db3da3678f780e8086b7734b72b/bundles/org.openhab.binding.lcn/src/main/resources/OH-INF/thing/thing-types.xml#L10